docs/database-restore.md

```diff
@@ -0,0 +1,38 @@
+# Apply for Postgraduate Teacher Training - Manual Deployment
+
+## Purpose
+
+This document describes the process of restoring the database from the Azure backups.
+
+Azure retains Postgres database backups for up to 35 days depending upon the configuration used. The Apply production environment has the full 35 days backup, the other environments have the retention period reduced to seven days.
+
+This document assumes that only the data in the database itself is at fault and the database server still exists.
+
+## Prerequisites
+
+To carry out a production database restore routine you will be required to have the following software installed:
+- PostgreSQL 9.6.x client (specifically `pg_dump` and `pg_restore`)
+
+## Instructions
+
+**NOTE: Before following the steps below you will need to request an elevation of your rights to the 'contributor' role through PIM in the Azure Portal if working on an app hosted in the test or production subscriptions. Guidance on PIM can be found in the [PIM Guide](pim-guide.md) document. PIM is not required in the development subscription.**
+
+These instructions assume that you are working on the production environment, but they apply equally to the other environments by substitution of the resource names prefix for the chosen environment.
+
+1. Launch the [Azure Portal](https://portal.azure.com).
```

Comment:
This is quite a long list and it's hard to see the logical steps. Could we break it up a bit? Proposed headings:

- Restore a backup to a temporary database instance
- Whitelist your IP
- Replace live data with backed up data
- Clean up